randovania/gui/preset_settings/elevators_tab.py

```diff
@@ -130,6 +130,8 @@ def _create_source_elevators(self):
         }
         self._elevator_source_for_location = copy.copy(checks)
         self._elevator_source_destination = {}
+        # todo: :D ???
+        teleporter_types = [self.game_description.dock_weakness_database.find_type("teleporter")]
```

Comment:
Somehow the game should decide what is considered a "teleporter thing"

Reply:
Consider this tab exclusively for Prime 1 and 2. Adjusting it to support other games is a task for the future and another PR.